artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -1108,6 +1109,11 @@ public void addHead(final MessageReference ref, boolean scheduling) {
       if (logger.isTraceEnabled()) {
          logger.trace("AddHead, size = {}, intermediate size = {}, references size = {}\nreference={}", queueMemorySize, intermediateMessageReferences.size(), messageReferences.size(), ref);
       }
+
+      if (RefCountMessage.isTraceEnabled()) {
+         RefCountMessage.deferredDebug(ref.getMessage(), "AddHead queue {}", this.getName());
```

Comment:
The others seem to be using more of a sentence structure, i.e "add head"

Edit...following the next comment, could alternatively go the other way, and make them all method-name structured.